services/user.js

```diff
@@ -200,14 +201,22 @@ class UserService {
 
     async getUser(params, context) {
         verifySession(context)
-            .verifyInitialized()
-            .verifyPermission(USER_PERMISSION_CONSTANTS.ADMIN.MANAGE_USER);
-        
+            .verifyInitialized();
+        await this.#isPermitted(context?.userInfo, USER_PERMISSION_CONSTANTS.ADMIN.MANAGE_USER);
+
         if (!params?.userID) {
             throw new Error(SUBMODULE_ERROR.INVALID_USERID);
         }
+        const isFederalLeadOnly = await this.#isManageUserScope(context?.userInfo, ROLES.FEDERAL_LEAD);
+        if (context?.userInfo?.role === ROLES.FEDERAL_LEAD && !isFederalLeadOnly) {
```

Comment:
Again, we should not check Federal Lead role.

Reply:
The Federal role is removed.